src/prune_builds

```diff
@@ -39,9 +41,14 @@ if os.path.isfile(builds_json):
 else:
     builddata = {'builds': []}
 
+# generate RFC3339 timestamp
+now = datetime.utcnow()
+rfc3339 = now.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Minor nit, feels weird to name a variable with an RFC ID, how about... `now = datetime.utcnow().strftime(...)` ?  Since nothing else uses `now` also.

Reply:
Also...hum, in rpm-ostree we called them ISO 8601 but I think they're the same?

Reply:
https://stackoverflow.com/a/522281

Looks like RFC 3339 is an extension of ISO 8601.  I opted for the former since that's what was requested by @wking.  I'll update the variable as suggested.

Reply:
> ... RFC 3339 is an extension of ISO 8601.

Your code here looks fine, but just to clarify: RFC3339 is a subset ISO 8601 which is [recommended for date-times on the internet][1], which is what this is.  Subsetting makes it easier to parse than generic ISO 8601.

[1]: https://tools.ietf.org/html/rfc3339